### PR TITLE
For #5126. Add F11 shortcut for full-screen view

### LINKED ISSUE #5126: F11 to Toggle Fullscreen
There's already a shortcut to toggle fullscreen (win+shift+enter) in uwp apps, but F11 is quicker way and a shortcut adopted in most of programs such as browsers, code editors , default file explorer etc.

### PR DESCRIPTION
**Pull request by levimatheri**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #5126 

**Details of Changes**
- Added F11 `KeyboardAccelerator` and implemented full-screen toggle command

**Validation**
How did you test these changes?
- [x] Built and ran the app

Animation(1)

### CODE REVIEW

File: Files/ViewModels/MainPageViewModel.cs [hunk 1]
```diff
@@ -159,6 +163,21 @@ private async void AddNewInstanceAccelerator(KeyboardAcceleratorInvokedEventArgs
...
+            {
+                view.ExitFullScreenMode();
+            }
+            else
+            {
+                view.TryEnterFullScreenMode();
+            }
+
+            e.Handled = true;
+        }
```
Comment (yaichenbaum) [inline Files/ViewModels/MainPageViewModel.cs:179]: Should this be in `MainPage.xaml.cs`?

Comment (yaichenbaum) [inline Files/ViewModels/MainPageViewModel.cs:179]: Should this be in `MainPage.xaml.cs`?

Comment (levimatheri) [reply Files/ViewModels/MainPageViewModel.cs:179]: Oops yeah. Will move to `MainPage.xaml.cs`

Comment (levimatheri) [reply Files/ViewModels/MainPageViewModel.cs]: @yair100 Should I move [`ICommand` property](https://github.com/files-community/Files/blob/a98b5f28707332ba54ecdbcdc0217a20d85609e0/Files/ViewModels/MainPageViewModel.cs#L51) to `MainPage.xaml.cs` as well?

Comment (yaichenbaum) [reply Files/ViewModels/MainPageViewModel.cs:179]: For now, yes. Eventually commands like this can be grouped into their own ViewModel so that we can access them from any page.

File: .github/FUNDING.yml [hunk 2]
```diff
@@ -1 +1 @@
-custom: ["paypal.me/yaichenbaum"]
+github: [yaichenbaum]
```
Comment (winston-de) [inline .github/FUNDING.yml:1]: Was this change intentional?

Comment (levimatheri) [reply .github/FUNDING.yml:1]: @winston-de I merged the from latest main and then did a squash commit. This is not part of my changes, must be from the last commit to main.

**Review by winston-de — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@levimatheri Great work!
